server/server/views/home.py

```diff
@@ -0,0 +1,15 @@
+from cornice.resource import resource, view
```

Comment:
Maybe I typed this comment earlier, nevermind...
We don't need this file at all, we don't make any requests to this right now